sources/packages/api/src/services/notification/gc-notify-actions.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from "@nestjs/common";
+import { Student } from "../../database/entities";
+import { getExtendedDateFormat, getPSTPDTDate } from "../../utilities";
+import { STUDENT_FILE_UPLOAD_TEMPLATE_ID } from "../../utilities/notification-utils";
+import { GCNotifyResult, RequestPayload } from "./gc-notify.model";
+import { GCNotifyService } from "./gc-notify.service";
+
+@Injectable()
+export class GCNotifyActionsService {
+  constructor(private readonly gcNotifyService: GCNotifyService) {}
+  private async fileUploadEmailPayload(
```

Comment:
Do we really need this method? Can it not be part of `sendFileUploadNotification`?

Reply:
No